### DISCUSSION TITLE
Selection not updating after control has loaded when not using valueAsObject and value = 0

### DISCUSSION THREAD

**Discussion by D4r1uZGND** [mskocik/svelecte#138] [Q&A] [answered]
First off I know I can solve this by using:
value={$inputVal} + bind:readSelection={$inputVal} + valueAsObject={true}

But would be nice to just have the basic case work out of the box as well
Example:
bind:value={$inputVal}
initial value for $inputVal is undefined where I want no default selection
On "edit-this" $inputVal gets assigned 0 for the first option in the select but handleValueUpdate's "if (passedVal)" skips this assignment (obv.)

Allowing another passthrough for "(passedVal)" => "(passedVal || passedVal > -1)" would cover index/string/array/object selections still but is there something simple I missed for the above example perhaps?

**Comment by mskocik** [comment, marked as answer]
I am not 100% sure, why there is only `if (passedVal)` check. There is no specific reason behind it. And your suggestion is okay and worth adding to handle the zero use-case

**Reply by D4r1uZGND** [reply]
Great, thanks for the quick answer :)
